src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -42,7 +42,6 @@ public override NoPia.EmbeddedTypesManager EmbeddedTypesManagerOpt
         private Dictionary<FieldSymbol, NamedTypeSymbol> _fixedImplementationTypes;
 
         private bool _needsGeneratedIsReadOnlyAttribute_Value;
-        private bool _needsGeneratedNullableAttribute_Value;
```

Comment:
>private bool _needsGeneratedNullableAttribute_Value; [](start = 8, length = 52)

What is the motivation behind this change?

Reply:
[Nullable] attribute is emitted regardless of whether there are nullable reference types.

---
In reply to: [171117119](https://github.com/dotnet/roslyn/pull/25087#discussion_r171117119) [](ancestors = 171117119)